src/rrule.ts

```diff
@@ -277,29 +407,36 @@ export default class RRule {
           if (value.indexOf(',') !== -1) {
             value = value.split(',')
             for (let j = 0; j < value.length; j++) {
-              if (/^[+-]?\d+$/.test(value[j])) value[j] = Number(value[j])
+              if (/^[+-]?\d+$/.test(value[j].toString())) {
+                value[j] = Number(value[j])
+              }
             }
           } else if (/^[+-]?\d+$/.test(value)) {
             value = Number(value)
           }
           key = key.toLowerCase()
+          // @ts-ignore
```

Comment:
I get really intensely curious when I see `@ts-ignore`'s, could we leave little notes about what we're avoiding with them?

Reply:
Didn't realize there were like 15 of these, maybe not a good use of time after all